tests/qos/qos_sai_base.py

```diff
@@ -556,6 +556,21 @@ def dutConfig(
 
             testPortIds = sorted(dutPortIps.keys())
 
+        elif tbinfo["topo"]["type"] == "t2":
+            for iface,addr in dut_asic.get_active_ip_interfaces(tbinfo).items():
+                if iface.startswith("Ethernet") and ("Ethernet-Rec" not in iface):
+                    portIndex = mgFacts["minigraph_ptf_indices"][iface]
+                    portIpMap = {'peer_addr': addr["peer_ipv4"]}
+                    dutPortIps.update({portIndex: portIpMap})
+                elif iface.startswith("PortChannel"):
+                    portName = next(
+                        iter(mgFacts["minigraph_portchannels"][iface]["members"])
+                    )
+                    portIndex = mgFacts["minigraph_ptf_indices"][portName]
+                    portIpMap = {'peer_addr': addr["peer_ipv4"]}
+                    dutPortIps.update({portIndex: portIpMap})
+            testPortIds = sorted(dutPortIps.keys())
+
```

Comment:
Can we have this logic similar to T1 but add an extra check for recycle ports ?

Reply:
![image](https://user-images.githubusercontent.com/46945843/189409413-565280fc-0956-4901-a75b-b0b81716d2a0.png)

Can we add "SUPPORTED_T2_TOPOS" as well ?

Reply:
@vmittal-msft  instead of supported t2 topos I am using topo type for all t2 so we do not need this change so we do not need to list each and every type of t2 topo here